wpcsite/jinja2.py

```diff
@@ -1,44 +1,24 @@
 """Helper functions for Jinja2 templates."""
 from __future__ import absolute_import
 
-from django.contrib.staticfiles.storage import staticfiles_storage
 from django.contrib.messages import get_messages
+from django.contrib.staticfiles.storage import staticfiles_storage
 from django.core.urlresolvers import reverse
-from django.utils.translation import ugettext, ungettext
-from jinja2 import Environment
 
-from webplatformcompat.helpers import env as wp_env
-from bcauth.helpers import env as bcauth_env
-from mdn.helpers import env as mdn_env
+from jinja2.ext import Extension
 
 
 def url(viewname, *args, **kwargs):
     return reverse(viewname, args=args, kwargs=kwargs)
 
 
-def environment(**options):
-    """Construct environment for jinja2 templates."""
-    env = Environment(
-        extensions=['jinja2.ext.i18n', 'jinja2.ext.autoescape'],
-        trim_blocks=True,
-        **options)
-    env.globals.update({
-        'static': staticfiles_storage.url,
-        'url': url,
-        '_': ugettext,
-        'gettext': ugettext,
-        'ngettext': ungettext,
-        'get_messages': get_messages,
-    })
-    for app_env in (wp_env, bcauth_env, mdn_env):
-        for group, to_add in app_env.items():
-            if group == 'globals':
-                for name, value in to_add.items():
-                    assert name not in env.globals
-                    env.globals[name] = value
-            else:
-                assert group == 'filters', 'Unknown group {}'.format(group)
-                for name, value in to_add.items():
-                    assert name not in env.filters
-                    env.filters[name] = value
-    return env
+def static(*args, **kwargs):
+    return staticfiles_storage.url(*args, **kwargs)
+
+
+class WPCExtension(Extension):
+    def __init__(self, environment):
+        super(WPCExtension, self).__init__(environment)
+        environment.globals['url'] = url
+        environment.globals['static'] = static
+        environment.globals['get_messages'] = get_messages
```

Comment:
I'm not sure myself. I think moving to webplatformcompat is the best of those options, as that is the closest equivalent we have to kuma. I'm also happy as-is, since I'm not sure what the convention should be.